sam3-program.bash

```diff
@@ -2,40 +2,40 @@
 
 LOG_FILE=/tmp/sam3-program.log
 
+ 
+for i in 23 19 18 20 17 27 22 4 
+do
+  if [ ! -d /sys/class/gpio/gpio$i ]
+  then
+    echo $i > /sys/class/gpio/export
+  fi
+done
+
 function super_reset()
 {
   #Set EM_Enable
-  echo 23 > /sys/class/gpio/export 
```

Comment:
Optional: The for cycle used above was nice. You could use a for here as well and the code will be easier to read.